### PR TITLE
Share node block

### PR DESCRIPTION
**Pull request by itkg-nanne**

### CODE REVIEW

File: ApiBundle/Controller/BlockController.php [hunk 1]
```diff
@@ -125,4 +125,27 @@ public function listSharedBlockByComponentAction($language, $component)
...
+     *
+     * @return FacadeInterface
+     */
+    public function shareBlockAction($blockId)
+    {
+        $this->denyAccessUnlessGranted(ContributionActionInterface::CREATE, BlockInterface::ENTITY_TYPE);
+
+        $repository = $this->get('open_orchestra_model.repository.block');
+        $block = $repository->find($blockId);
+        $block->setTransverse(true);
```
Comment (alavieille) [inline ApiBundle/Controller/BlockController.php]: checj if block is different null

File: ApiBundle/Controller/BlockController.php [hunk 2]
```diff
@@ -125,4 +125,27 @@ public function listSharedBlockByComponentAction($language, $component)
...
+     * @param string  $blockId
+     *
+     * @Config\Route("/share/block/{blockId}", name="open_orchestra_api_block_share")
+     * @Config\Method({"PUT"})
+     *
+     * @return FacadeInterface
+     */
+    public function shareBlockAction($blockId)
+    {
+        $this->denyAccessUnlessGranted(ContributionActionInterface::CREATE, BlockInterface::ENTITY_TYPE);
```
Comment (alavieille) [inline ApiBundle/Controller/BlockController.php]: i'm not sur for role what is the role to create transverse block

Comment (itkg-nanne) [reply ApiBundle/Controller/BlockController.php]: this one! (come from Admin/BlockController)

**Comment by itkg-nanne** [comment]
update done @alavieille

**Comment by itkg-nanne** [comment]
added contribution right. update done @alavieille

File: BackofficeBundle/Resources/public/template/Block/blockView._tpl.html [hunk 3]
```diff
@@ -6,6 +6,9 @@
         <% } %>
         <ul class="toolbar pull-right">
             <% if (false === block.get('transverse')) { %>
+                <% if (user.access_section.contribution) { %>
```
Comment (alavieille) [inline BackofficeBundle/Resources/public/template/Block/blockView._tpl.html]: not contribution but configuration

File: ApiBundle/Controller/BlockController.php [hunk 4]
```diff
@@ -125,4 +127,41 @@ public function listSharedBlockByComponentAction($language, $component)
...
+     * @return FacadeInterface
+     *
+     * @throws UnexpectedValueException
+     * @throws ClientAccessDeniedHttpException
+     */
+    public function shareBlockAction($blockId)
+    {
+        $this->denyAccessUnlessGranted(ContributionActionInterface::CREATE, BlockInterface::ENTITY_TYPE);
+
+        if (!$this->getUser()->hasRole(ContributionRoleInterface::DEVELOPER) &&
```
Comment (alavieille) [inline ApiBundle/Controller/BlockController.php]: sorry i check voter and it's correct so this lines are useless

**Comment by itkg-nanne** [comment]
update done @alavieille

**Comment by alavieille** [comment]
:+1:

**Pull request merged**